recipe/meta.yaml

```diff
@@ -1,31 +1,32 @@
 {% set name = "ctypesgen" %}
-{% set version = "1.1.2.dev4+g8a62d02" %}
+{% set version = "1.1.1a" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-pypdfium2-team
   version: {{ version }}
 
 source:
-  url: file:///Users/jolivieri/repos/upstream/ctypesgen-pypdfium2-team/dist/ctypesgen-1.1.2.dev4+g8a62d02.tar.gz
-  sha256: d23910fd783b8eb89d30cbb51b9c1bbedcc4143a50617c69959da4951cac94f9
+  git_url: https://github.com/pypdfium2-team/{{ name }}.git
+  git_rev: 8a62d02a6fd350df6d22e179bcd164df0da770d4
```

Comment:
maybe worth noting that the commit is from https://github.com/pypdfium2-team/ctypesgen/tree/pypdfium2 branch, so next person will know where to look at if there is a release or tag ?

Reply:
The commit is on the master branch it must have been fast forwarded to that commit of that branch, but it's still on the master branch. Which was also used when `pypdfium2` was built.